src/css/homepage.css

```diff
@@ -1,3 +1,48 @@
+/*Homepage fonts*/
+h1 {
+	font-family: "Open Sans", Arial, sans-serif;
+	font-size: 25px;
+	color: white;
+}
+
+h3 {
+	font-family: "Permanent Marker", Helvetica, sans-serif;
+	font-size: 40px;
+	color: #38023B;
+	font-weight: bold;
+}
+
+h4 {
+	font-family: "Permanent Marker", Helvetica, sans-serif;
+	font-size: 30px;
+	color: white;
+	font-weight: bold;
+	padding-right: 60px;
+}
+
+/* Homepage video CSS */
+
+.homepage-video {
+    object-fit: cover;
+    width: 100%;
+    height: 50em;
+    margin-top: -10em;
+}
+
+/* Homepage hero-text CSS */
+.hero-text {
```

Comment:
See my comments for this class on the HTML, we need to follow the name convention for classes to get more context out of it.